aas_core_meta/v3_1.py

```diff
@@ -3442,14 +3443,9 @@ def __init__(
                     and len(self.specific_asset_IDs) >= 1
             )
         )
-    ) or (
-            self.entity_type != Entity_type.Self_managed_entity
-            and self.global_asset_ID is None
-            and self.specific_asset_IDs is None
     ),
     "Constraint AASd-014: Either the attribute global asset ID or "
-    "specific asset ID must be set if entity type is set to self-managed entity. "
-    "They are not existing otherwise."
+    "specific asset ID must be set if Entity/entityType is set to :attr:`Entity_type.Self_managed_entity`."
 )
```

Comment:
Can someone please pay additional attention to my adaptation of the invariant? Does this do what it says it does?